sdk/CHANGELOG.md

```diff
@@ -19,6 +19,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Security -->
 
+## 1.0.3 - 2023-MM-DD
+
+### Added
+
+- `last_sync` field to `ClientInner` instead of a local static ref;
+
```

Comment:
Or maybe actually also add `- Clients returning the default protocol parameters when multiple Client instances are used;` in a fixed section?

Reply:
Yeah, that